doc/rst/technotes/extern.rst

```diff
@@ -222,8 +222,8 @@ called on Chapel strings that are stored on the same locale; calling
 
 .. note::
 
-  ``c_string`` is expected to be deprecated in a future release in favor
-  of instead using ``c_ptr`` types such as ``c_ptrConst(c_char)``.
+  ``c_string`` is deprecated in favor of ``c_ptrConst(c_char)``, which can be
+  used with corresponding functions such as ``c_ptrToConst``.
```

Comment:
If we can, it would be nice to put the link for more information (https://chapel-lang.org/docs/main/language/evolution.html#version-1-32-september-2023) in this warning. 

Maybe something like:
```
 ``c_string`` is deprecated in favor of ``c_ptrConst(c_char)``.
  See :ref:`c_string deprecation <readme-evolution.c_string-deprecation>`
  for more information.
```

I think this would require putting a target like `.. _readme-evolution.c_string-deprecation:` in `evolution.rst` just above the note on c_string deprecation.

Reply:
Oh, I wasn't aware of that info on the language evolution page, will definitely link to it.